src/components/Navigation.js

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import { NavLink } from 'react-router-dom';
+
+const Navbar = () => {
+  const links = [
+    {
+      id: 1,
+      path: '/math-magicians/',
+      text: 'Home',
```

Comment:
- [ ] Kindly make sure to set the `home` page to be the default page which will be rendered once we start the application. You can do that by setting the `path` to `/`.